### PR TITLE
[FEATURE] Bootstrap theme docs

### PR DESCRIPTION
**Pull request by marikaris**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits
- [ ] Added Feature/Fix to release notes

### CODE REVIEW

File: docs/developer_documentation/creating-themes.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+@import url("//fonts.googleapis.com/css?family=IBM+Plex+Sans");
+```
+After doing this you can use the font throughout your bootstrap theme. Selecting the font as base
+font can be done in the "Typography" tab of your theme builder.
+
+## Bootstrap 4
+Creating bootstrap 4 themes can be done using this tool:
+[bootstrap.build](https://bootstrap.build/app).
+
+### Fonts
```
Comment (ConnorStroomberg) [inline docs/developer_documentation/creating-themes.md]: maybe combine with font section on b3-3

File: docs/developer_documentation/creating-themes.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+To see which colors a website of your customer uses, open the console of your browser and select the
+element with the preferred color. Alternatively, for chrome use the "eye dropper color picker" 
+plugin. However please note colors selected by the color picker might be a slightly bit different 
+from the original. The color picker is especially handy when selecting colors from images, rather
+than HTML elements.
+
+### Exporting
+Download at least the unminimized css file. This is the readable version of your css and it can be 
+loaded in molgenis using the theme manager. If you created a theme for a specific project, put it in
+the molgenis-projects repository on github. It could also be nice to export the .less files for
```
Comment (ConnorStroomberg) [inline docs/developer_documentation/creating-themes.md]: 'could' , would

File: docs/developer_documentation/creating-themes.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+**
+Molgenis allows for many customizations. One of them is uploading your own bootstrap themes.
+**
+
+# Bootstrap themes
+
+Molgenis uses currently two versions of bootstrap: bootstrap 3.3.7 and bootstrap 4.x. To customize
```
Comment (ConnorStroomberg) [inline docs/developer_documentation/creating-themes.md]: grammatically incorrect sentence, should be; 'Currently Molgenis uses ....

File: docs/developer_documentation/creating-themes.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+#### 2. Set the same colors
+Set your own colors, or copy either the colors of bootstrap 3 of bootstrap 4 to both themes in the
+"Color" section. Bootstrap 4 has an additional secondary color, which is not in bootstrap 3, here
+you can use the info color, or another if you prefer. 
+
+#### 3. The menu
+To make the menu look the same, use your browsers debugger to compare the bootstrap 3 and 4 theme
+and copy css styling between the two until they look the same. Do the same alterations in your
+bootstrap css. Some of the alterations you want to do cannot be done in the theme builders. Paste
+this css code well documented on top of you bootstrap theme file. You might want to use the
```
Comment (ConnorStroomberg) [inline docs/developer_documentation/creating-themes.md]: 'Paste
this css code well documented on top of you bootstrap theme file. You might want to use the ' seems a bit vage/general to me , i suggest to remove this line

File: docs/developer_documentation/creating-themes.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+#### 3. The menu
+To make the menu look the same, use your browsers debugger to compare the bootstrap 3 and 4 theme
+and copy css styling between the two until they look the same. Do the same alterations in your
+bootstrap css. Some of the alterations you want to do cannot be done in the theme builders. Paste
+this css code well documented on top of you bootstrap theme file. You might want to use the 
+```!important``` in some cases, but be careful with that.
+
+#### 4. Don't mind the sign in and sign out button
+These buttons have such different classes between the bootstrap 3 and 4 molgenis menu, that you
```
[the remark this answers is not in the document]
Comment (marikaris) [reply docs/developer_documentation/creating-themes.md]: It's just the sign in and out button, we messed up the classes. They are totally different between bootstrap 3 and 4 and we switch the class if you sign in or sign out

**Pull request merged**